docs/EditTutorial.md

```diff
@@ -479,7 +479,7 @@ import * as React from "react";
 import { Edit, SimpleForm, TextInput, SelectInput } from "react-admin";
 
 export const BookEdit = () => (
-  <Edit>
+  <Edit emptyWhileLoading>
```

Comment:
I think you should revert the changes in this file. We don't need to introduce this prop, which is for corner case, in a general presentation of the Edit logic.